policy/testdata/annotations/data.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
/lgtm too

just a comment, we could lean on acceptance tests([tests/](https://github.com/open-policy-agent/conftest/tree/master/tests)) instead of trying to write tests against `engine.go`. 

Apart from that, I think the engine_test.go needs a general refactor because its tests look like e2e rather than unit tests, and not testing the engine's own functionality like Test_Check or Test_Runtime or Test_Query